### PR TITLE
[Accessibility] Fix typos in inline documentation in woocommerce client, includes, and lib folders

### PR DESCRIPTION
**Pull request by Manussakis**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Fixes part of the #37502 issue.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

NA.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

**Comment by Manussakis** [comment]
@dkotter This PR is ready for code review.

File: plugins/woocommerce/client/legacy/js/jquery-blockui/jquery.blockUI.js [hunk 1]
```diff
@@ -191,7 +191,7 @@
 			// enable if you want key and mouse events to be disabled for content that is blocked
 			bindEvents: true,
 
-			// be default blockUI will supress tab navigation from leaving blocking content
+			// be default blockUI will suppress tab navigation from leaving blocking content
```
Comment (dkotter) [inline plugins/woocommerce/client/legacy/js/jquery-blockui/jquery.blockUI.js]: Seems like there's another typo here, `be` for `by`?

```suggestion
			// by default blockUI will suppress tab navigation from leaving blocking content
```

**Review by dkotter — APPROVED** [review]
Since this is just typo fixes, this can bypass our QA and move to the next step

File: plugins/woocommerce/includes/wc-update-functions.php [hunk 2]
```diff
@@ -416,7 +416,7 @@ function wc_update_209_brazillian_state() {
 
 	// phpcs:disable WordPress.DB.SlowDBQuery
 
-	// Update brazillian state codes.
+	// Update brazilian state codes.
```
Comment (Konamiman) [inline plugins/woocommerce/includes/wc-update-functions.php]: Shouldn't this be uppercased? (so `Brazilian`)

Comment (Manussakis) [reply plugins/woocommerce/includes/wc-update-functions.php]: Good catch, @Konamiman , thanks!
Fixed: [Uppercase the Brazilian word](https://github.com/woocommerce/woocommerce/pull/50739/commits/ac30fa22aad8dfe65dfe30310af4a889a5e50217)

**Review by Konamiman — APPROVED** [review]

**Pull request merged**